app/code/community/Drip/Connect/Helper/Data.php

```diff
@@ -119,19 +119,35 @@ public function isSubscriberExists($email, $storeId = null)
     /**
      * check if customer exists on the website
      *
+     * @param string $email
+     * @param Drip_Connect_Model_Configuration $config
+     *
      * @return bool
      */
-    public function isCustomerExists($email, $websiteId = null)
+    public function isCustomerExists($email, $config)
```

Comment:
Might be nice to use type hinting here:
```suggestion
    public function isCustomerExists($email, Drip_Connect_Model_Configuration $config)
```